src/storage/Cargo.toml

```diff
@@ -49,6 +49,7 @@ risingwave_rpc_client = { path = "../rpc_client" }
 # ], optional = true }
 serde = { version = "1", features = ["derive"] }
 smallvec = "1"
+spin = "0.5.2"
```

Comment:
spin 0.5.2 seems too old (current version = 0.9.3), can its version be updated?

Reply:
Emmmm, I just changed it because other library depend on 0.5.2